### PR TITLE
[BUGFIX] Renderer for UnexpectedRowsExpectation

### PR DESCRIPTION
**Pull request by tyler-hoffman**
Here's what it looks like:

Screenshot 2024-12-12 at 10 12 31 AM

- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, visit our [community resources](https://docs.greatexpectations.io/docs/core/introduction/community_resources#contribute-code-or-documentation).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: great_expectations/expectations/core/unexpected_rows_expectation.py [hunk 1]
```diff
@@ -133,6 +140,37 @@ def _prescriptive_template(
...
+        **kwargs,
+    ) -> list[RenderedStringTemplateContent]:
+        runtime_configuration = runtime_configuration or {}
+        styling = runtime_configuration.get("styling")
+        params = substitute_none_for_missing(
+            configuration.kwargs,  # type: ignore[union-attr]
+            ["unexpected_rows_query"],
+        )
+
+        template_str = "Unexpected rows query: $unexpected_rows_query"
```
Comment (NathanFarmer) [inline great_expectations/expectations/core/unexpected_rows_expectation.py:161]: How does this look when the query is multi-line? I suspect it's multi-line more often than not. Adding a newline is probably better IMO.

**Review by NathanFarmer — CHANGES_REQUESTED** [review]

File: great_expectations/expectations/core/unexpected_rows_expectation.py (hunk 1 above)
Comment (tyler-hoffman) [reply great_expectations/expectations/core/unexpected_rows_expectation.py:161]: Do you just mean a newline after the `:`?

Comment (tyler-hoffman) [reply great_expectations/expectations/core/unexpected_rows_expectation.py:161]: Either way I'll manually test and post a screenshot with multiline

Comment (NathanFarmer) [reply great_expectations/expectations/core/unexpected_rows_expectation.py:161]: Yes that's what I mean

**Review by NathanFarmer — APPROVED** [review]

**Pull request merged**